core/lib/spree/encryptor.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Spree
+  # Spree::Encryptor is a thin wrapper around ActiveSupport::MessageEncryptor.
+  class Encryptor
+    # @param key [String] the 256 bits signature key
+    def initialize(key)
+      @crypt = ActiveSupport::MessageEncryptor.new(key)
+    end
+
+    # Encrypt a value
+    # @param value [String] the value to encrypt
+    # @return [String] the encrypted value
+    def encrypt(value)
+        @crypt.encrypt_and_sign(value)
```

Comment:
The body of the encrypt and decrypt methods is indented 2 spaces too far.

Reply:
Good catch, my bad fixed. Thank you 😄.